experiments/templates/experiments/new.html

```diff
@@ -8,25 +8,34 @@
     {% trans 'experiments:new:header' %} - {{ block.super }}
 {% endblock %}
 
+{% block html_head %}
+<script nonce="{{ request.csp_nonce }}">
```

Comment:
Missing ``{{ block.super }}``

Not required atm, but in the future omitting that might cause problems